sdk/search/azure-search-documents/azure/search/documents/indexes/models/__init__.py

```diff
@@ -34,16 +34,15 @@
 from . import _edm
 from ..._generated.models import SuggestOptions
 from .._generated.models import (
-    SearchAlias,
-    AzureMachineLearningSkill,
     AnalyzeResult,
     AnalyzedTokenInfo,
     AsciiFoldingTokenFilter,
-    AzureOpenAIEmbeddingSkill,
-    AzureOpenAIParameters,
-    AzureOpenAIVectorizer,
+    BlobIndexerDataToExtract,
```

Comment:
There seem to be a lot of new model classes exposed in the public API. Should any of these be mentioned in the CHANGELOG?

Reply:
Added.